requirements/basic.txt

```diff
@@ -11,4 +11,5 @@ shapely>=2.0
 pydantic>=1.10.0
 PyYAML
 dask
-toml
\ No newline at end of file
+toml
+dask
```

Comment:
Where is dask used? Is it a jax requirement?

Reply:
dask is an xarray dependency. Toml is for pyproject.toml.  I'm not sure why this PR seems to add dask into basic.txt though, it seems it is already in develop and [pre/1.9.0](https://github.com/flexcompute/tidy3d/blob/pre/1.9.0/requirements/basic.txt). I'll look into it.